### ISSUE TITLE
'OutEdgeView' object has no attribute 'sort'

### ISSUE THREAD

**Issue by MihaelaMia** [djordon/queueing-tool#47]
No longer compatible with NetworkX? I'm running OSX 10.11.6 and python 2.7.13, it's the same with python3.
This is my code:
```
import queueing_tool as qt  
import networkx as nx  
g = qt.generate_random_graph(20, seed=3)
```

And this is the crash:
```
Traceback (most recent call last):
  File "example.py", line 3, in <module>
    g = qt.generate_random_graph(200, seed=3)
  File "/usr/local/lib/python2.7/site-packages/queueing_tool/graph/graph_generation.py", line 121, in generate_random_graph
    g = minimal_random_graph(num_vertices, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/queueing_tool/graph/graph_generation.py", line 225, in minimal_random_graph
    g = QueueNetworkDiGraph(g.to_directed())
  File "/usr/local/lib/python2.7/site-packages/queueing_tool/graph/graph_wrapper.py", line 243, in __init__
    edges.sort()
AttributeError: 'OutEdgeView' object has no attribute 'sort'
```

**Comment by djordon** [comment]
Sorry for the delay in getting back to you. Would you mind posting the output of `pip freeze` here?

**Comment by djordon** [comment]
This issue is related to the recently released version of networkx (v2.0), where some breaking changes were introduced. I'll be pushing out a fix for this shortly.

**Comment by djordon** [comment]
This was fixed with https://github.com/djordon/queueing-tool/pull/45. Reinstalling queueing-tool, with the latest version (v1.2.1), should address this issue for you. Let me know if anything else comes up.
